src/plant/Chamomile.java

```diff
@@ -0,0 +1,9 @@
+package plant;
```

Comment:
Let's make the root package for example. com.flower.kiev.
And put all flower-classes into the package entity or domain.